pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   shelf_static: '^0.2.0'
   shelf_web_socket: '^0.0.1'
   source_span: '^1.0.0'
-  stack_trace: '^1.2.0'
+  stack_trace: '^1.2.1'
```

Comment:
Generally I don't worry that much about bugs that are fixed in patch versions; `1.2.0` isn't so much incompatible as it is completely broken. But it is nice to have `pub downgrade` work, so sure.